flow/connectors/connelasticsearch/elasticsearch.go

```diff
@@ -174,13 +176,20 @@ func (esc *ElasticsearchConnector) SyncQRepRecords(ctx context.Context, config *
 					bulkIndexErrors = append(bulkIndexErrors,
 						fmt.Errorf("id:%s type:%s reason:%s %s", item.DocumentID, res.Error.Type,
 							res.Error.Reason, causeString))
+					if res.Error.Type == "illegal_argument_exception" &&
+						strings.Contains(res.Error.Reason, "Number of documents in the index can't exceed") {
+						bulkIndexHasFatalError = true
```

Comment:
```suggestion
						bulkIndexFatalError = res.Error
```
Then change type to `error`. Allows for better logging at end